app/agents/voice/automatic/stt/__init__.py

```
@@ -224,6 +225,29 @@ def get_stt_service(voice_name: Optional[str] = None):
             params=soniox_params,
             vad_force_turn_endpoint=config.SONIOX_VAD_FORCE_TURN_ENDPOINT,
         )
+    elif config.STT_PROVIDER == "elevenlabs":
+        if not config.ELEVENLABS_API_KEY:
+            raise ValueError(
+                "ELEVENLABS_API_KEY is required when STT_PROVIDER=elevenlabs"
+            )
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)